app/tests/system/nightwatch-config.js

```diff
@@ -1,9 +1,10 @@
 var path = require('path');
 var appPath = path.join(process.cwd(), '../ios/build/Build/Products/Debug-iphonesimulator/scaffold.app');
```

Comment:
We should be sensitive to which build type our tests run under: `Release` or `Debug`.  I'd think I'd rather we do our testing under a `Release` flavour.